src/x/sync/pooled_worker_pool_test.go

```diff
@@ -74,16 +80,14 @@ func TestPooledWorkerPoolGoWithTimeout(t *testing.T) {
 	for i := 0; i < workers*2; i++ {
 		result := p.GoWithTimeout(func() {
 			wg.Wait()
-		}, 10*time.Millisecond)
+		}, 100*time.Millisecond)
```

Comment:
This was just me reverting a time change since originally I was seeing if that was the issue